src/common/hydrogenic.jl

```diff
@@ -0,0 +1,22 @@
+@doc raw"""
+Radial functions from solutions of Hydrogenic Schrödinger equation.
+Same as Wannier90 user guide Table 3.3.
+# Arguments
+- `r`: radial grid
+- `n`: principal quantum number
+- `α`: diffusivity, ``\frac{Z}/{a}`` where ``Z`` is the atomic number and
+    ``a`` is the Bohr radius.
+"""
+function radial_hydrogenic(r::AbstractVector{T}, n::Integer, α::Real=1.0) where {T<:Real}
```

Comment:
This function is not type stable. Even if `T` is `Float32`, this function will return a `Float64` because `(3/2)` automatically gives a Float64. One way to solve this is to use the pattern `(3/T(2))` for example.